index.html

```diff
@@ -0,0 +1,347 @@
+                    <p>javascript</p>
+                </div>
+            </div>
+
+            <div class="project-button-link-container">
+                <a href="">
+                    See project
+                </a>
+            </div>
+        </div>
+
+        <div class="project-card">
+            <div class="project-image">
+                <img src="assets/images/project3.png" alt="">
+            </div>
+
+            <div class="project-description">
+                <h1 class="project-title">Multi-Post <br> Stories</h1>
+                <div class="details">
+                    <p>CANOPY </p>
+                    <div class="cercle"></div>
+                    <p>Back End Dev</p>
+                    <div class="cercle"></div>
+                    <p>2015</p>
+                </div>
+
+                <p class="detail-description">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+
+                <div class="used-programming-languages">
+                    <p>html</p>
+                    <p>css</p>
+                    <p>javascript</p>
+                </div>
+            </div>
+
+            <div class="project-button-link-container">
+                <a href="">
+                    See project
+                </a>
+            </div>
+        </div>
+
+        <div class="project-card">
+            <div class="project-image">
+                <img src="assets/images/project4.png" alt="">
+            </div>
+
+            <div class="project-description">
+                <h1 class="project-title">Multi-Post <br> Stories</h1>
+                <div class="details">
+                    <p>CANOPY </p>
+                    <div class="cercle"></div>
+                    <p>Back End Dev</p>
+                    <div class="cercle"></div>
+                    <p>2015</p>
+                </div>
+
+                <p class="detail-description">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+
+                <div class="used-programming-languages">
+                    <p>html</p>
+                    <p>css</p>
+                    <p>javascript</p>
+                </div>
+            </div>
+
+            <div class="project-button-link-container">
+                <a href="">
+                    See project</a>
+            </div>
+        </div>
+    </section>
+
+    <section class="about-section">
+        <div class="info-description">
+            <h1>About Myself</h1>
+            <p class="">Hello I’m a software developer! I can help you build a product , feature or website. Look
+                through some of my work and experience! If you like what you see and have a project you need coded,
+                don’t hesitate to contact me.
+            </p>
+        </div>
+
+        <div class="contact">
+            <a href="">LET’S CONNECT</a>
```

Comment:
- [ ] KIndly remove this `a` tag and make it `h2,..` tag for semantically reasons